src/extractor/scripting_environment_lua.cpp

```diff
@@ -34,7 +34,12 @@ namespace
 template <class T>
 auto get_value_by_key(T const &object, const char *key) -> decltype(object.get_value_by_key(key))
 {
-    return object.get_value_by_key(key, "");
+    auto v = object.get_value_by_key(key);
```

Comment:
Do I need to adjust the C++ formatting here? Not sure how the tidy workflow works.